src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -52,6 +52,8 @@
     private Connection connection;
     private AppConfig appConfig;
     private Boolean cassandraAliasing;
+    private Integer minReturnedMetrics = 0;
```

Comment:
I don't think we can modify this during runtime, should we make the attribute `final` (and set in the constructor like we're currently doing)? If we do, we'll have to handle the case when the option is not set (`Integer.MAX_VALUE` or `-1`?).